tests/unit/test_schemas_json_load.py

```diff
@@ -238,3 +238,41 @@ def test_valid_rights(val, expected):
         {'rights': [{'lang': 'en'}]}
     )
     assert 'lang' not in errors
+
+
+@pytest.mark.parametrize(('access_right', 'valid'), [
+    ('open', True),
+    ('embargoed', True),
+    ('restricted', True),
+    ('closed', True),
+    ('FOO', False)
+])
+def test_access_right(access_right, valid):
+    data, errors = (
+        MetadataSchemaV1(partial=True).load({'access_right': access_right})
+    )
+
+    if valid:
+        assert not errors
+        assert data['access_right'] == access_right
+    else:
+        assert 'access_right' in errors
+
+
+@pytest.mark.parametrize(('access', 'valid'), [
+    ({}, False),
+    ({'metadata_restricted': True}, False),
+    ({'files_restricted': False}, False),
+    ({'metadata_restricted': True, 'files_restricted': False}, True),
```

Comment:
Moderate:

``` python
 ({'metadata_restricted': True, 'files_restricted': False}, True)
```

Isn't this an invalid case? how can the record be restricted but the files not?

Reply:
Haha, this was just a random selection of True/False! I wasn't thinking about the meaning but just about the rules of the schema. Indeed this scenario should not occur like we talked about. Since this will be changed anyway, I am not too worried. I will change both to True though to be clear.